### PR TITLE
viewport settings

### PR DESCRIPTION
**Pull request by dmitrii-z**
### PURPOSE
Implemented a limited number of settings for the render settings dialog, viewport only

### CODE REVIEW

**Comment by ARozghon** [comment]
will be approved after autotests will be completed

**Comment by ARozghon** [comment]
Approved by QA

File: pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py [hunk 1]
```diff
@@ -696,6 +696,24 @@ def houdini_parm_name(name):
                 'defaultValue': False
             }
         ]
+    },
+    {
+        'name': 'ViewportSettings',
+        'houdini': {},
+        'settings': [
```
Comment (bsavery) [inline pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py:703]: Why would we set this to false?  Does it ever not work.

**Review by V-Gorash — APPROVED** [review]

**Comment by ARozghon** [comment]
new information: Need to check HybridPro settings with the viewport render.
please, not merge to the master without my approve

**Comment by bsavery** [comment]
> new information: Need to check HybridPro settings with the viewport render. please, not merge to the master without my approve

@ARozghon any update.

**Comment by bsavery** [comment]
Ok so from @ARozghon we have issues still on Hybrid Pro.  Please fix before we can merge

**Comment by ARozghon** [comment]
approved by manual checking. Waiting for the auto jobs results

**Comment by ARozghon** [comment]
Approved

**Pull request merged**